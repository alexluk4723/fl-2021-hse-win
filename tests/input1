b:=^nm^|^k^
a:=b
c:=a
return c
