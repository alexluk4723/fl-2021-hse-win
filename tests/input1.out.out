graph graphname {
b [label="head:b"]
b -- b_3
b_3 [label="|"]
b_1 [label="nm"]
b_2 [label="k"]
b_3 -- b_1
b_3 -- b_2
a [label="head:a"]
a -- a_1
a_1 [label="b"]
c [label="head:c"]
c -- c_1
c_1 [label="a"]
MAIN__ [label="MAIN" color=red]
MAIN__ -- c
}
